eng/native/build-commons.sh

```diff
@@ -94,6 +94,10 @@ build_native()
         cmakeArgs="-DCMAKE_SYSTEM_VARIANT=MacCatalyst $cmakeArgs"
     fi
 
+    if [[ "$targetOS" == "iOSSimulator" || "$targetOS" == "tvOSSimulator" ]]; then
+	cmakeArgs="-DCMAKE_SYSTEM_VARIANT=$targetOS $cmakeArgs"
```

Comment:
nit: spaces for indentation:
```suggestion
        cmakeArgs="-DCMAKE_SYSTEM_VARIANT=$targetOS $cmakeArgs"
```